### PR TITLE
wip: more questions

### PR DESCRIPTION
**Pull request by siosTomBeckenhauer**

### CODE REVIEW

**Comment by travisgoldie** [comment]
Does this allow us to judge:

"Does this candidate have the needed skills, personality, experience, and work ethic to build SIOS iQ?"

**Comment by siosTomBeckenhauer** [comment]
Skills? Yes, we work in JS, CSS, HTML.

File: single-page-app-exercise/QUESTIONS.md [hunk 1]
```diff
@@ -5,8 +5,54 @@
...
+- Q: How can you achieve immutability in your own code? 
+- Q: Explain the difference between synchronous and asynchronous functions. 
+
+## General Questions:
+- Q: What did you learn yesterday/this week? 
+- Q: What excites or interests you about coding? 
+- Q: What is a recent technical challenge you experienced and how did you solve it? 
+- Q: If you could master one technology this year, what would it be? 
+- Q: What UI, Security, Performance, SEO, Maintainability or Technology considerations do you make while building a web application or site? 
+- Q: Can you describe the difference between progressive enhancement and graceful degradation?
```
Comment (travisgoldie) [inline single-page-app-exercise/QUESTIONS.md:33]: Maybe to dependent on definitions.

**Comment by travisgoldie** [comment]
Agreed we do. However, if we only ask these questions, can we judge if a candidate can produce some working code in our AngularJS app within the 2nd week?

Do we feel the candidate will be able to understand how AngularJS works well enough to build directives, create templates, update configs, create providers, etc.?

Write unit tests for code?

**Comment by siosTomBeckenhauer** [comment]
Agreed, make a PR.

**Comment by travisgoldie** [comment]
Maybe I am reading this wrong, but it looks like your change deleted all the existing lines. Not sure why

**Comment by travisgoldie** [comment]
Never mind. Your changes are in the middle.

**Comment by travisgoldie** [comment]
Looks good to me

**Pull request merged**